### PR TITLE
New blog: writing fast constraint streams

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

**Review by rsynek — APPROVED** [review]
LGTM, nice article

**Comment by ge0ffrey** [comment]
Looks good. Do not merge - I 'll figure out tomorrow how to align the jbake branch

File: blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+:awestruct-tags: [constraint, insight, performance]
+
+Do you want OptaPlanner to run faster?
+Do you want to increase your score calculation speed, reaching great solutions sooner?
+Let me show you how to optimize your
+https://docs.optaplanner.org/latest/optaplanner-docs/html_single/index.html#constraintStreams[Constraint Streams]
+constraints for performance and scalability.
+Turns out you only need to remember one advice:
+
+== DO LESS
```
Comment (ge0ffrey) [inline blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc:14]: Personal opinion: I wouldn't write all caps. "Do less" works just as fine. All caps is shouting.

File: blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc [hunk 2]
```diff
@@ -0,0 +1,193 @@
+= Writing fast constraints with OptaPlanner: The secret recipe
```
Comment (ge0ffrey) [inline blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc:1]: The JBake migration is almost done, but getting it to publish automatically might take a few days. LEt's get this out first and I 'll deal with merge conflicts on my end (same for Jiri's PR, I 'll merge it tomorrow)

Let's put the date on 25 or 26 May (adjust the file name) and merge(=publish) it.

**Pull request merged**